shared-bindings/wiznet/wiznet5k.c

```diff
@@ -136,6 +160,7 @@ STATIC mp_obj_t wiznet5k_ifconfig(size_t n_args, const mp_obj_t *args) {
         return mp_obj_new_tuple(4, tuple);
     } else {
         // set
+        // XXX should this automatically stop DHCP here?
```

Comment:
What are the XXX comments for? Can you either remove them or rephrase them as `TODO: <explanation of what to do?`? Thanks!